.github/workflows/test-python.yaml

```diff
@@ -32,8 +32,16 @@ jobs:
 
           pip install -r cmd/earlystopping/medianstop/v1beta1/requirements.txt
 
+      - name: Generate Tensorflow Event file
+        run: |
+          LOG_PATH=test/unit/v1beta1/metricscollector/testdata/tfevent-metricscollector/logs
+          python examples/v1beta1/trial-images/tf-mnist-with-summaries/mnist.py --epochs 10 --batch-size 200 --log-path $LOG_PATH
```

Comment:
Do we want to add this generation to the test file (`test_tfevent_metricscollector.py`) ?
I think it would be much easier to run this unit test locally if we have everything in one place.
WDYT @tenzen-y ?

Reply:
I think we would better not call `tf-mnist-with-summaries/mnist.py` by `test_tfevent_metricscollector.py` since we should not make `test_tfevent_metricscollector.py` dependent on example code. Although as you say, I think it is inconvenient to run unit-test for python on our local machine so I think it might be better to add some commands to Makefile or to create a shell script like the below.
What do you think? @andreyvelich 

```make
prepare-pytest:
  pip install -r test/unit/v1beta1/requirements.txt
  ...

prepare-pytest-testdata:
  <Generate Tensorflow Event file>
  ...

pytest: prepare-pytest prepare-pytest-testdata
  pytest ./test/unit/v1beta1/suggestion
  ...
```

Reply:
I see, it makes sense. Before changing this what do others think @gaocegege @johnugeorge ?
Does it look good if we add test log generator to the Makefile ?

Reply:
LGTM. Should we add the generated file to .gitignore?

Reply:
Sure, I think we should add the log files to `.gitignore`, thanks! @gaocegege

Reply:
I addressed this at https://github.com/kubeflow/katib/commit/8cc2df5f7b9689bf2cc7d5057b59f1d261591a76.